### PR TITLE
barebones ticket verification workflow

### PR DESCRIPTION
**Pull request by bobqywei**
Still uses hardcoded addresses until QR code is fully built out. Tested using existing NFT transaction on Mumbai. Also fixed routing.

### CODE REVIEW

**Comment by rohilG** [comment]
we should add yarn.lock to .gitignore

**Comment by bobqywei** [comment]
> we should add yarn.lock to .gitignore

Someone already did in master

**Review by loganwatchorn — APPROVED** [review]

File: src/views/Verifier.js [hunk 1]
```diff
@@ -1,204 +1,142 @@
...
+    const transfers_to_creator = await Moralis.Web3API.account.getNFTTransfers(
+      moralis_options
     );
-    if (!response.ok) {
-      console.log("Network response not OK!");
+    console.log("fetched transactions to " + creator_address);
+
+    const nfts_from_attendee = transfers_to_creator.result
+      .filter((transaction) => transaction.from_address === attendee_address)
+      .map((transaction) => transaction.token_address + transaction.token_id);
```
Comment (rohilG) [inline src/views/Verifier.js:36]: if we don't put a space, comma or something between these or store them as tuples, we might have some issues decoupling them.

(ofc we do know the exact length of an address but that would be kinda messy)

File: src/views/Verifier.js [hunk 2]
```diff
@@ -1,204 +1,142 @@
...
+    transfers_to_creator.result
+      .filter(
+        (transaction) =>
+          transaction.from_address ===
+            "0x0000000000000000000000000000000000000000" &&
+          moralisDateToUnixTimestamp(transaction.block_timestamp) >= mint_date
+      )
+      .forEach((transaction) =>
+        nfts_minted_by_creator.add(
+          transaction.token_address + transaction.token_id
```
Comment (rohilG) [inline src/views/Verifier.js:49]: same here

File: src/views/Verifier.js (hunk 1 above)
Comment (rohilG) [reply src/views/Verifier.js:36]: nm this is just for a boolean value. could still be nice to add something between for debugging purposes down the line

**Review by rohilG — APPROVED** [review]

**Pull request merged**